.travis.yml

```diff
@@ -1,61 +0,0 @@
-language: generic
-dist: xenial
-services:
-  - xvfb
-
-addons:
-  apt:
-    packages:
-    # Qt dependencies
-    - libglu1-mesa-dev
-    - libxkbcommon-x11-0
-    - libxcb-icccm4
-    - libxcb-image0
-    - libxcb-keysyms1
-    - libxcb-randr0
-    - libxcb-render-util0
-    - libxcb-xinerama0
-    - pulseaudio
-    - libpulse-mainloop-glib0
-    # Wx dependencies
-    - libsdl2-2.0-0
-
-env:
-  global:
-    - INSTALL_EDM_VERSION=3.0.1
-      PYTHONUNBUFFERED="1"
-
-matrix:
-  include:
-    - env: RUNTIME=3.6 TOOLKITS="null"
-    - env: RUNTIME=3.6 TOOLKITS="pyqt5 pyside2"
-    - env: RUNTIME=3.6 TOOLKITS="wx"
-  allow_failures:
-    - env: RUNTIME=3.6 TOOLKITS="wx"
-  fast_finish: true
-
-cache:
-  directories:
-    - "~/.cache"
-
-before_install:
-  - mkdir -p "${HOME}/.cache/download"
-  - if [[ ${TRAVIS_OS_NAME} == 'linux' ]]; then ./install-edm-linux.sh; export PATH="${HOME}/edm/bin:${PATH}"; fi
-  - edm install -y wheel click coverage
-install:
-  - for toolkit in ${TOOLKITS}; do edm run -- python etstool.py install --runtime=${RUNTIME} --toolkit=${toolkit} || exit; done
-  - if [[ ${TRAVIS_EVENT_TYPE} == 'cron' ]]; then for toolkit in ${TOOLKITS}; do edm run -- python etstool.py install --runtime=${RUNTIME} --toolkit=${toolkit} --source || exit; done; fi
-  # etsdemo
-  - pushd ets-demo && for toolkit in ${TOOLKITS}; do edm run -- python etstool.py install --runtime=${RUNTIME} --toolkit=${toolkit} || exit; done && popd
-script:
-  - if [[ ${TOOLKITS} == 'null' ]]; then edm run -- python etstool.py flake8 --runtime=${RUNTIME} --toolkit=null; fi
```

Comment:
we also now don't use the ci command `python etstool.py flake8` anywhere. However we still flake8 on PRs in the style-checks workflow with `python -m flake8`

Reply:
I think the `flake8` ci command is still useful for local development.